master/internal/resourcemanagers/config.go

```diff
@@ -0,0 +1,39 @@
+package resourcemanagers
```

Comment:
I would rename this file and maybe the config objects in it to indicate that they only exists for backwards compatibility.

Reply:
Talked offline. Will rename the file to be `deprecated_config.go` and keep the name of `Config` as is.